src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
Don't forget about the encapsulation and the access modifiers. Our variables here should be `private`.

Also, not really a mistake, but consider making better variable names for simplicity and clarity. Simple `color` and `number` variable names would suffice as they are already in the context of the `Ball` class.